stl/inc/ranges

```diff
@@ -6058,7 +6058,11 @@ namespace ranges {
 
         constexpr explicit slide_view(_Vw _Range_, const range_difference_t<_Vw> _Count_) noexcept(
             is_nothrow_move_constructible_v<_Vw>) /* strengthened */
-            : _Range(_STD move(_Range_)), _Count{_Count_} {}
+            : _Range(_STD move(_Range_)), _Count{_Count_} {
+#if _CONTAINER_DEBUG_LEVEL > 0
+            _STL_VERIFY(_Count > 0, "The window size must be positive (NFIXME [range.slide.view]/1)");

```

Comment:
This `FIXME` must be corrected before checkin.

Reply:
https://isocpp.org/files/papers/N4917.pdf is now available.